scripts/gui/thrive_style.css

```diff
@@ -1643,6 +1687,39 @@ div::-webkit-scrollbar-thumb:hover {
   font-size: 85%;
 }
 
+#ColourWheel {
+  position: absolute;
+  width: 200px;
+  height: 200px;
+  left: 15px;
+  top: 350px;
+  border-radius: 50%;
+  background-size: 200px 200px;
+  background-image: url("../../Textures/gui/colour_wheel.png");
```

Comment:
This is more like a note to myself than a review comment: this file needs to be added to svn.

Reply:
Should be in svn now.